docroot/modules/custom/forex_rates/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "custom/forex_rates",
+    "type": "drupal-drush",
+    "extra": {
+        "drush": {
+            "services": {
+                "drush.services.yml": "^10"
+            }
+        }
+    }
+}
```

Comment:
we don't require this.

Reply:
I was exploring ,module packaging and how it works with Composer.
It is also scaffolded from drush command. I can remove if not required.